app/src/app/pages/home/randomizer-page/item-pool-settings/item-pool-settings.component.ts

```diff
@@ -1,5 +1,5 @@
 import { RandomConsumableMode } from 'src/app/entities/enum/randomConsumableMode';
-import { FormGroup } from '@angular/forms';
+import { FormControl, FormGroup } from '@angular/forms';
```

Comment:
Can you cleanup the unused imports from the recent changes? I know I'm not one to talk since there are unused imports all over the place but it would at least make the PR merge cleaner, with only actually modified files listed.

Reply:
LGTM aside from that

Reply:
> Can you cleanup the unused imports from the recent changes? I know I'm not one to talk since there are unused imports all over the place but it would at least make the PR merge cleaner, with only actually modified files listed.

Done. I usually rely on a keyboard shortcut to cleanup my imports, but on the existing files it kept introducing extraneous changes by rearranging existing imports and formatting unrelated blocks, so I tried to remove them manually instead and missed a couple.